src/fuzz/dummy_fuzz_test.cc

```diff
@@ -0,0 +1,148 @@
+    google::cloud::spanner::Instance instance("emulator", "emulator");
+    google::cloud::spanner::InstanceAdminClient instance_client(
+        google::cloud::spanner::MakeInstanceAdminConnection(
+            emulator_connection));
+
+    future<StatusOr<google::spanner::admin::instance::v1::Instance>>
+        instance_future = instance_client.CreateInstance(
+            google::cloud::spanner::CreateInstanceRequestBuilder(instance,
+                                                                 "emulator")
+                .SetDisplayName("emulator")
+                .SetNodeCount(1)
+                .SetLabels({{"label-key", "label-value"}})
+                .Build());
+    StatusOr<google::spanner::admin::instance::v1::Instance>
+        status_or_instance = instance_future.get();
+    if (!status_or_instance) {
+      throw std::runtime_error(status_or_instance.status().message());
+    }
+    std::cout << "Created instance [" << instance << "]\n";
+
+    // Then we create a simple database.
+    google::cloud::spanner::Database database(instance, "test-db");
+
+    auto admin_client =
+        DatabaseAdminClient(google::cloud::spanner::MakeDatabaseAdminConnection(
+            emulator_connection));
+    future<StatusOr<google::spanner::admin::database::v1::Database>> f =
+        admin_client.CreateDatabase(database, {R"""(
+          CREATE TABLE Singers (
+              SingerId   INT64 NOT NULL,
+              FirstName  STRING(1024),
+              LastName   STRING(1024),
+              SingerInfo BYTES(MAX)
+          ) PRIMARY KEY (SingerId))""",
+                                               R"""(
+          CREATE TABLE Albums (
+              SingerId     INT64 NOT NULL,
+              AlbumId      INT64 NOT NULL,
+              AlbumTitle   STRING(MAX)
+          ) PRIMARY KEY (SingerId, AlbumId),
+              INTERLEAVE IN PARENT Singers ON DELETE CASCADE)"""});
+    StatusOr<google::spanner::admin::database::v1::Database> db = f.get();
+    if (!db) throw std::runtime_error(db.status().message());
+    std::cout << "Created database [" << database << "]\n";
+
+    google::cloud::spanner::Client client(
+        google::cloud::spanner::MakeConnection(database, emulator_connection));
+
+    std::string query = std::string("INSERT INTO Singers (FirstName) VALUES (") + std::string((char*)Data, Size) + std::string(")");
+
+    client.ExecuteQuery(
+        google::cloud::spanner::SqlStatement(query)
+    );
+    // auto rows = client.ExecuteQuery(
+    //     google::cloud::spanner::SqlStatement("SELECT 'Hello World'"));
+
+    // for (auto const& row :
+    //      google::cloud::spanner::StreamOf<std::tuple<std::string>>(rows)) {
+    //   if (!row) throw std::runtime_error(row.status().message());
+    //   std::cout << std::get<0>(*row) << "\n";
+    // }
+
+    return 0;
+  } catch (std::exception const& ex) {
+    std::cerr << "Standard exception raised: " << ex.what() << "\n";
+    return 1;
```

Comment:
We may want to reach out to Sneha about which exceptions might be expected to be thrown by the client.  It's possible that for this use case, we'll want to abort/exit on some exceptions with an appropriate message that the exception is not expected/not allowed.

Short term, this is fine though.